### ISSUE TITLE
[Question] Is there a way to resize splits using the keyboard?

### ISSUE THREAD

**Issue by abemilk** [onivim/oni2#2505]
I used to do this in vim and it was pretty useful.

**Comment by glennsl** [comment]
Yes, there are some resize commands you can use either through the Command Palette, which is really cumbersome, or through keybindings. There are some [existing keybindings](https://github.com/onivim/oni2/blob/b6e2388115cc08a47888c9312fa3e7fae6102ed0/src/Store/KeyBindingsStoreConnector.re#L379-L416), but they're broken because of bugs in the keybinding engine, so I recommend setting up your own keybindings. Refer to the [available commands here](https://onivim.github.io/docs/configuration/key-bindings#window-management).
